chia/rpc/wallet_rpc_api.py

```diff
@@ -362,6 +364,11 @@ async def get_network_info(self, request: Dict):
         address_prefix = self.service.config["network_overrides"]["config"][network_name]["address_prefix"]
         return {"network_name": network_name, "network_prefix": address_prefix}
 
+    async def push_tx(self, request: Dict):
```

Comment:
let's add a check that we have a full node connection, and return error message in the case we don't have it.